rocky/tools/forms/ooi_form.py

```diff
@@ -100,6 +100,18 @@ def generate_form_fields(
             else:
                 fields[name] = forms.CharField(max_length=256, **default_attrs)
 
+        # ruff: noqa: ERA001
+        # Currently we are not ready to use the following line as the
+        # event manager is not aware of the deletion of a generic OOI
+        # it does work for 'end-point'-OOIs like MutedFinding and the
+        # field is hidden for now
+        # fields["end_valid_time"] = forms.DateTimeField(
+        #     label="Expires by",
+        #     widget=forms.DateTimeInput(attrs={"type": "datetime-local"}),
+        #     required=False,
+        # )
+        fields["end_valid_time"] = forms.DateTimeField(widget=forms.HiddenInput(), required=False)
```

Comment:
A hidden field with an empty value doesn't really seem useful?

Reply:
Try removing it and see how functionality breaks 😉

The field is necessary because the OOI itself does not have an `end_valid_time` field, so we have to add it manually here (otherwise it wont be aggregated from the form [editor note lol] to the view, also on the specialized) -- and also comment above it.

Reply:
There is no second endpoint that handles the OOI for muted findings, and as such its the same endpoint for creating regular OOI's as for MuteOOIs. For the later, we can 'safely' introduce this end_valid_time, since they are endpoints in the graph (for now), but for generic OOI's the form can at least be  made ready but inert.